Design-System/styles/utils.css

```diff
@@ -0,0 +1,69 @@
+/* *******************Utility classes for cart********************* */
+body {
+    background-color: #eee;
+}
+.address_header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+.address-mgmt{
+    max-width: 30ch;
+}
+
+.header_row input {
+    width: 20%;
+}
+.col {
+    background-color: transparent !important;
+}
+.coupon_box {
+    border: 1px solid;
+}
+
+.displayFlexRow{
+    display: flex;
+    flex-direction: row;
```

Comment:
this is redundant code, displat flex has row direction by default, no need to mention it

Reply:
read class name for making it sense so anybody can used  it that class is utility class

Reply:
Yes class name you can keep that, but no one is going to see CSS right, so that's why its redundant